lib/prelude.nat

```diff
@@ -43,3 +43,25 @@ unit °C = celsius
 
 conv x:K ⇒ °C = (x / K)°C - 273.15°C
 conv x:°C ⇒ K = (x/°C + 273.15)K
+
+unit joule = kg * (m / s)^2
+unit J = joule
+
+unit angstrom = 100 pm
+unit ångström = angstrom
+unit Å = angstrom
+
+unit micron = μm
+
+c = 299792458
+
+unit astronomical_unit = 149597870700 m
+unit AU = astronomical_unit
+unit au = astronomical_unit
+
+unit lightsecond = c m
```

Comment:
Given the changed definition of `c`, this should be 
```suggestion
unit lightsecond = c * 1 s
```
which seems a lot more sane to me :thinking:

Reply:
👍